File: RMI_реализация/rmiclient/src/operation/ServiceOperation.java
package operation;

import java.rmi.Remote;
import java.rmi.RemoteException;
import java.util.List;
import types.Service;

/**
 *
 * @author Lactea Via
 */
public interface ServiceOperation extends Remote{
    List<Service> getListOfService()throws RemoteException;
    List<Service> addNewService(Service item)throws RemoteException;
    int getSumOfService()throws RemoteException;
}

File: serverH/server/ServiceOperation.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace serverH
{
    public interface ServiceOperation
    {
        List<Service> getListOfService();
        List<Service> addNewService(Service item);
        List<Service> delService(int index);
        int getSumOfService();

    }
}
